Color/Colors.php

```diff
@@ -167,150 +167,162 @@ class Colors
      *
      * @return array
      */
-    public static function getAll()
+    public static function getAll($isInverted = true)
```

Comment:
It is, but for extension. Can we consider this BC-break ok? I'm not even sure people are using this class directly, so I doubt they extend it. If you think otherwise, then the only solution I see is creating a new method and deprecating this one.